File: ABC079/A/Main.java
import java.io.*;
import java.util.*;

class Main{
    public void run() throws Exception{
        Scanner cin = new Scanner(System.in);
        String arg = cin.next();

        if(arg.contains("111") || arg.contains("222") ||arg.contains("333") || arg.contains("444")||arg.contains("555") || arg.contains("666")||arg.contains("777") || arg.contains("888") || arg.contains("999") || arg.contains("000")){
            System.out.println("Yes");
        } else {
            System.out.println("No");
        }
    }
    public static void main(String[] args) throws Exception{
        new Main().run();
    }
}

File: ABC079/A/Main.kt
import java.util.*
fun main(args: Array<String>){
    val cin = Scanner(System.`in`)
    /* print(cin.nextInt()+cin.nextInt()) */
    /* print(" ")                         */
    if(cin.next().contains("111") || cin.next().contains("222") ||cin.next().contains("333") || cin.next().contains("444")||cin.next().contains("555") || cin.next().contains("666")||cin.next().contains("777") || cin.next().contains("888") || cin.next().contains("999") || cin.next().contains("000")){
        println("Yes")
    } else {
        println("No")
    }
}

